security/advancedtls/examples/go.mod

```diff
@@ -5,5 +5,12 @@ go 1.15
 require (
 	google.golang.org/grpc v1.33.1
 	google.golang.org/grpc/examples v0.0.0-20201020200225-9519efffeb5d
-	google.golang.org/grpc/security/advancedtls v0.0.0-20201020200225-9519efffeb5d
+	google.golang.org/grpc/security/advancedtls v0.0.0-20201102215344-15ae9fc2b247
 )
+
+// TODO(Zhenlian): remove these lines when advancedtls becomes stable.
```

Comment:
I think you should just remove this line altogether.  It's best to know these examples are in sync with HEAD for the whole repo.